charts/k8ssandra/values.yaml

```diff
@@ -405,7 +405,7 @@ reaper:
     # -- Specifies the container repository for cassandra-reaper
     repository: docker.io/thelastpickle/cassandra-reaper
     # -- Tag of an image within the specified repository
-    tag: 2.2.0
+    tag: 2.2.1
```

Comment:
There was a related change in reaper-operator that @burmanm made the other day going to 2.2.0, I believe it's only for testing purposes but let's cover that as well:

https://github.com/k8ssandra/reaper-operator/pull/37

Reply:
kk, will push a PR there as well.